tentang kami.html

```diff
@@ -0,0 +1,652 @@
+                                <li><a href=""
+                                        class="py-4 px-2 text-black font-semibold hover:text-green-500 transition duration-300">Perusahaan</a>
+                                </li>
+                                <div class="flex items-center">
+                                    <li>
+                                        <div class="dropdown inline-block relative">
+                                            <button
+                                                class=" text-black hover:text-green-500 font-semibold py-2 px-4 rounded inline-flex items-center">
+                                                <span class="mr-1">Lainnyaaa</span>
+                                                <svg class="fill-current h-4 w-4" xmlns="http://www.w3.org/2000/svg"
+                                                    viewBox="0 0 20 20">
+                                                    <path
+                                                        d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z" />
+                                                </svg>
+                                            </button>
+                                            <ul class="dropdown-menu absolute hidden text-gray-700 pt-1">
+                                                <li class=""><a
+                                                        class="rounded-t bg-gray-200 hover:bg-gray-400 py-2 px-4 block whitespace-no-wrap"
+                                                        href="#">One</a></li>
+                                                <li class=""><a
+                                                        class="bg-gray-200 hover:bg-gray-400 py-2 px-4 block whitespace-no-wrap"
+                                                        href="#">Two</a></li>
+                                                <li class=""><a
+                                                        class="rounded-b bg-gray-200 hover:bg-gray-400 py-2 px-4 block whitespace-no-wrap"
+                                                        href="#">Three is the magic number</a></li>
+                                            </ul>
+                                        </div>
+                                    </li>
+                                </div>
+                        </div>
+                    </div>
+                    <!-- Navbar items kanan -->
+                    <div class="hidden md:flex items-center space-x-6 ">
+                        <!-- <div class="flex items-center">
+                            <a href=""
+                                class="py-4 px-2 text-black font-semibold hover:text-green-500 transition duration-300">ID
+                            </a>
+                            <img class="" src="image/donw.png" alt=""> -->
+
+                        <div class="dropdown inline-block relative">
+                            <button
```

Comment:
untuk tag button, biasakan button ada attirbute typenya. Semisal `type="button"`

Reply:
sudah mas tapi belum tau itu bener tidak

Reply:
Untuk pengunaan type="submit" ini digunakann semisal ada di dalam tag <form>, karena ngga ada tag form bisa dibuat tpye="button". 

Lebih detail bisa pelajari referenssi berikut
https://www.tutorialspoint.com/What-is-difference-between-button-vs-input-type-button

Reply:
Baik mas